$OpenBSD: patch-src_smtp_cpp,v 1.1 2019/04/15 17:24:49 cwen Exp $

powerpc and arm have unsigned char by default, fix: 
error: narrowing conversion of '-1' from 'int' to 'char' inside { }

Index: src/smtp.cpp
--- src/smtp.cpp.orig
+++ src/smtp.cpp
@@ -279,7 +279,7 @@ void Smtp::receivedAuth(QString &responseLine)
 static const char base64char[65] =
 "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 
-static const char base64val[128] = {
+static const signed char base64val[128] = {
 	-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
 	-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
 	-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 62, -1, -1, -1, 63,
